Compilation failure with SwaggerUi

## Describe the bug

```
error[E0277]: the trait bound `AxumRouter<_>: std::convert::From<SwaggerUi>` is not satisfied
   --> crates/runtime/src/http/routes.rs:176:20
    |
176 |             .merge(SwaggerUi::new("/docs").url("/docs/openapi.json", get_api_doc()));
    |              ----- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::convert::From<SwaggerUi>` is not implemented for `AxumRouter<_>`
    |              |
    |              required by a bound introduced by this call
    |
    = help: the trait `From<SwaggerUi>` is not implemented for `AxumRouter<_>`
            but trait `From<axum_extra::routing::Resource<_>>` is implemented for it
    = help: for that trait implementation, expected `axum_extra::routing::Resource<_>`, found `SwaggerUi`
    = note: required for `SwaggerUi` to implement `Into<AxumRouter<_>>`
note: required by a bound in `AxumRouter::<S>::merge`
   --> /Users/kevin/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/axum-0.7.9/src/routing/mod.rs:232:12
    |
230 |     pub fn merge<R>(self, other: R) -> Self
    |            ----- required by a bound in this associated function
231 |     where
232 |         R: Into<Router<S>>,
    |            ^^^^^^^^^^^^^^^ required by this bound in `Router::<S>::merge`
help: consider removing this method call, as the receiver has type `SwaggerUi` and `SwaggerUi: std::convert::From<SwaggerUi>` trivially holds
    |
176 -             .merge(SwaggerUi::new("/docs").url("/docs/openapi.json", get_api_doc()));
176 +             .merge(SwaggerUi::new("/docs"));
```

## To Reproduce

`make install-dev`

--- a/.github/workflows/makefile_targets.yml
+++ b/.github/workflows/makefile_targets.yml
@@ -68,6 +68,10 @@ jobs:
         run: |
           make install
 
+      - name: make install-dev
+        run: |
+          make install-dev
+
       - name: make install-with-models
         run: |
           make install-with-models

--- a/crates/runtime/Cargo.toml
+++ b/crates/runtime/Cargo.toml
@@ -144,7 +144,7 @@ tract-core = "0.21.10"
 url = "2.5.4"
 util = { path = "../util" }
 utoipa = { version = "5", features = ["axum_extras"], optional = true }
-utoipa-swagger-ui = { version = "9", features = ["axum"], optional = true }
+utoipa-swagger-ui = { version = "8", features = ["axum"], optional = true }
 
 uuid = { workspace = true, features = ["v7"] }
 workers = { path = "../workers" }

